specifyweb/specify/api.py

```diff
@@ -115,7 +115,20 @@ def resource_dispatch(request, model, id):
                             content_type='application/json')
 
     elif request.method == 'DELETE':
-        delete_resource(request.specify_user_agent, model, id, version)
+        recordsetid = request.GET.get('recordsetid', None)
```

Comment:
Did you test this? I don't think `request.GET` should contain the request parameters if the  method is DELETE.

In any case, I don't think the REST verb DELETE on the resource should be overloaded for record-set-item deletion. It should be done by issuing DELETE on the record-set-item resource: `DELETE /api/specify/recordsetitem/id/`. That would require knowing the record-set-item id. For that, recommend extending the `record_set_info` metadata that gets added to the main resource api to include the record-set-item id.

Reply:
This is actually a bug in the audit log code because RecordSetItem doesn't have a version field. It looks like `recordversion` in the audit log cannot be null, so perhaps it should be record as zero if `obj` doesn't have a version attribute. OTOH, should changes to record sets even be recorded in the audit log?